docs/api/visitors-get-visitors.md

```diff
@@ -1,17 +1,77 @@
-# GET Visitors
+# Get a List of all Visitors
 
-### URL
+## `GET` /visitors
 
 ```
 {server_url}/visitors
 ```
 
-### Parameters
+View information from all visitor types.
 
-### Request headers
+### Headers
 
-### Request body
+None
 
-### Return body
+### Path Parameters
 
-### Return status
+None
+
+### Query Parameters
+
+| Name | Data Type | Required/Optional | Description |
+| --- | --- | --- | --- |
+| visitor_id | Integer | Required | The visitor type ID number (positive only) |
+| visitor_type | String | Required | A term for the visitor profile type |
+| habitat | String | Required | The preferred habitat |
```

Comment:
Maybe we can clarify that this is "the preferred habitat **for the visitor**", to avoid confusion. (Adding one comment for this feedback to be concise, but this feedback would apply to the other places in the doc where we use the same phrasing.)